src/routes/blog/spyder-6-daniel/+page.md

```diff
@@ -0,0 +1,102 @@
+
+## Review work and project management
+
+Much of Daniel's responsibilities involve reviewing, mentoring and assisting with the work of other developers, an important role in any project.
+Daniel helped [Ryan Clary](https://github.com/mrclary), the developer who created a new set of installers for Spyder 6 using [Conda-Constructor](https://conda.github.io/constructor/), which are more powerful, flexible and updateble, and support installing packages, environments and Spyder plugins locally and remotely, as needed for the new remote development functionality among others.
+In addition, he tested and validated the installers' update process, as well as finding and reporting several bugs on Windows.
+
+Daniel also reviewed and assisted with [Angela Remolina](https://github.com/AngelaRemolina)'s work on major improvements to Spyder's File/Symbol Switcher.
+Angela, an intern who worked for on Spyder over several months, migrated the File/symbol switcher to a plugin, so it is modular and accessible to other plugins.
+As part of her project he helped manage, she also added lightning-fast, [ripgrep](https://github.com/BurntSushi/ripgrep)-accelerated searching of all files in the current project.
+
+Finally, along with [Juan-Sebastian Bautista](https://github.com/jsbautista) whom he has mentored and managed, Daniel performed an in-depth review of the numerous UI/UX enhancements included in Spyder 6.
+This work has made Spyder much easier and more intuitive to use, including for users with accessibility needs and with translated versions.
+Thanks to Daniel's mentorship, Juan-Sebastian looks to continue his work with Spyder into the future.<br><br>
+
+
+![Spyder release page, showing a list of version numbers and their corresponding information and files](daniel-release-page.png "Daniel has released dozens of recent Spyder versions")
+
+## Spyder release management
+
+Since February 1, 2023, thanks to the CZI EOSS 5 grant funding, Daniel has also been active as Spyder's Release Manager, taking on the substantial task of handling the many critical steps required to produce stable packages for the many platforms we support, including:
+
+* [PyPI](https://pypi.org/project/spyder/) (pip)
+* [Conda-Forge](https://anaconda.org/conda-forge/spyder) (Conda)
+* [Spyder installers](https://www.spyder-ide.org/download/), and the
+* [GitHub releases](https://github.com/spyder-ide/spyder/releases).
+
+Since then, he has served as Release Manager for one minor and eight bugfix releases for Spyder 5 (from 5.4.3 to 5.5.6), and eight pre-releases, one major release and three bugfix releases for Spyder 6 (from 6.0 alpha2 to 6.0.3).
+Without his work, our users wouldn't have been able to benefit from all the testing and stable releases that we have, especially for Spyder 6!
+
+
+## What's next?
+
+In the immediate future, Daniel will continue his critical role as the Release Manager for Spyder, ensuring users continue to see regular stable updates for Spyder 6.
```

Comment:
```suggestion
In the immediate future, Daniel will continue his critical role as the Release Manager for Spyder, ensuring users keep seeing regular updates for Spyder 6.
```

* Replace `continue` by `keep` because `continue` is used before.
* `stable` seems redundant to me.